Celeste.Mod.mm/Mod/Registry/DecalRegistry.cs

```diff
@@ -162,15 +169,40 @@ public static class DecalRegistry {
             }},
         };
 
-        // Helper functions for scaling Decal Registry fields
+        /// <summary>
+        /// Adds a custom property to the decal registry. See <see cref="PropertyHandlers"/> for the list of Everest-defined properties.
+        /// </summary>
+        public static void AddPropertyHandler(string propertyName, Action<Decal, XmlAttributeCollection> action) {
+            if (PropertyHandlers.ContainsKey(propertyName)) {
```

Comment:
what happens if a mod wants to patch an existing property handler, eg. to turn off all parallax decals? could this warning be optionally silenced?

Reply:
that's exactly why we want this warning in place. someone overriding Everest/another mod's property affects any mods that use it so it's not recommended and we don't want to do that silently